### PR TITLE
added invoice page 

### PR DESCRIPTION
**Pull request by pnaik73**
please review and merge

### CODE REVIEW

**Review by ankitarevankar — COMMENTED** [review]
check and fix it

**Review by ankitarevankar — COMMENTED** [review]
check and update.

File: ComcastCRMGUIFramework/src/test/java/practice/test/InvoicePage.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package practice.test;
+
+public class InvoicePage {
+	//Hello ankita
+
+}
```
Comment (ankitarevankar) [inline ComcastCRMGUIFramework/src/test/java/practice/test/InvoicePage.java:7]: do changes in xpath.

**Review by ankitarevankar — COMMENTED** [review]
completed

**Comment by pnaik73** [comment]
fixed please merge

**Pull request merged**